### ISSUE TITLE
Script in Xcode scheme no immediate feedback in Xcode <sad face>

### ISSUE THREAD

**Issue by rob-nash** [realm/SwiftLint#2145]
### New Issue Checklist

- [x ] Updated SwiftLint to the latest version
- [x ] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Bug Report

So I had no problems when the script was a build phase, on the target. However, I moved the script onto an Xcode scheme and now I don't get immediate feedback in Xcode.

When moving the script to a scheme, swiftlint was failing. To get it working, I had to explicitly specify the path of the config file. Which is fair enough.I also had to set 'include' in the config to specify the project root. Ok no worries there.

### Command
```bash
exec > ${PROJECT_DIR}/prebuild_swiftlint.log 2>&1
echo "SwiftLint Log"

if which swiftlint >/dev/null; then
swiftlint --config ${PROJECT_DIR}/.swiftlint.yml
else
echo "warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint"
fi
```

### Output

Contents of prebuild_swiftlint.log

```
SwiftLint Log
Loading configuration from '/Users/robertnash/Code/WelshDanceOrgansation/DanceUK/.swiftlint.yml'
Linting Swift files in current working directory
Linting 'Dequeable.swift' (1/68)
Linting 'DequeableCollectionView.swift' (2/68)
Linting 'DequeableComponentIdentifiable.swift' (3/68)
Linting 'DequeableTableView.swift' (4/68)
LOADS OF PATHS HERE
Done linting! Found 234 violations, 29 serious in 68 files.
```

Done linting! but there is no feedback in Xcode anymore. It looks like there is one additional step needed.

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.25.0
* Installation method used (Homebrew, CocoaPods, building from source, etc)? Homebrew
* Paste your configuration file:

```yml
included:
  - ${PROJECT_DIR}
excluded: # paths to ignore during linting. Takes precedence over `included`.
  - Carthage
reporter:
  - Xcode

# configurable rules can be customized from this configuration file
# binary rules can set their severity level
line_length: 190
force_cast: warning # implicitly
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
No
* Which Xcode version are you using (check `xcode-select -p`)?
Version 9.3 (9E145)

**Comment by marcelofabri** [comment]
What exactly do you mean by moving the script to a scheme?

**Comment by rob-nash** [comment]
The following script can be 1) a build phase on a target 2) an 'action' on a scheme

```
exec > ${PROJECT_DIR}/prebuild_swiftlint.log 2>&1
echo "SwiftLint Log"

if which swiftlint >/dev/null; then
swiftlint --config ${PROJECT_DIR}/.swiftlint.yml
else
echo "warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint"
fi
```

[Ref1](https://stackoverflow.com/questions/13065905/scheme-script-vs-build-phase-script)
[Ref2](https://gist.github.com/acrookston/55d69a16cd5363426dbf7a3d6a9ee6ce)

**Comment by rob-nash** [comment]
I just moved the script back as a build phase :)

**Comment by BryanVision** [comment]
I need this working as a scheme script as client wants SwiftLint on Sandbox but not Production scheme and it doesn't appear to be using the default code in the docs.  Is there a way to get this working as a scheme script?
